app/views/hyrax/base/_relationships_parent_row.html.erb

```diff
@@ -1,6 +1,5 @@
-<tr>
-  <th><%= t(".label", type: type.humanize) %></th>
-  <td>
+<dt> <%= t(".label", type: type.humanize) %> </dt>
+<dd>
```

Comment:
Similar comment here about the unordered list inside of a description list. The relationships aren't key/value pairs and could be just an unordered list. It's a little different than metadata.